ui/src/app/preferences/views/SSHKeys/SSHKeyFormFields/SSHKeyFormFields.js

```diff
@@ -0,0 +1,136 @@
+import { useSelector } from "react-redux";
+import PropTypes from "prop-types";
+import React from "react";
+
+import { formikFormDisabled } from "app/settings/utils";
+import { sshkey as sshkeySelectors } from "app/preferences/selectors";
+import { useFormikErrors } from "app/base/hooks";
+import Col from "app/base/components/Col";
+import Form from "app/base/components/Form";
+import FormCardButtons from "app/base/components/FormCardButtons";
+import FormikField from "app/base/components/FormikField";
+import Link from "app/base/components/Link";
+import Notification from "app/base/components/Notification";
+import Row from "app/base/components/Row";
+import Select from "app/base/components/Select";
+import Textarea from "app/base/components/Textarea";
+
+export const SSHKeyFormFields = ({ editing, formikProps }) => {
+  const saving = useSelector(sshkeySelectors.saving);
+  const saved = useSelector(sshkeySelectors.saved);
+  const errors = useSelector(sshkeySelectors.errors);
+  useFormikErrors(errors, formikProps);
+  let sshkeyErrors;
+  if (errors) {
+    if (typeof sslkeyErrors === "string") {
```

Comment:
This should be `typeof errors === "string"`

Reply:
Oops.